satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -791,10 +848,11 @@ datasets:
   # ---- CSR products ------------
   nir39:
     name: nir39
+    long_name: TOA Brightness Temperature at 3.9um
+    standard_name: toa_brightness_temperature
     key: '#4#brightnessTemperature'
     resolution: 48006.450653072
     wavelength: [3.48, 3.92, 4.36]
-    standard_name: toa_brightness_temperature
     units:  "W/sr-1/m-2"
```

Comment:
I'm quite sure that the CSR for IR channels is expressed in brightness temperatures? In that case the units shall be is K.

Reply:
done